### PR TITLE
feat: implement vault metadata

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/utils/getVaultsData.ts [hunk 1]
```diff
@@ -56,14 +64,21 @@ const fetchAndFormatData = async (
...
   owner: Address | undefined;
   name: string | undefined;
   relays: RelayData;
   jobs: JobData;
   tokens: TokenData[];
+  description: string;
 }> => {
   const vaultContract = { address: vaultAddress, abi: vaultABI };
+
+  // TODO
```
Comment (turtlemoji) [inline src/utils/getVaultsData.ts:78]: for later?

File: src/utils/getVaultsData.ts [hunk 2]
```diff
@@ -123,11 +139,23 @@ const fetchAndFormatData = async (
     tokensData = formatTokensData(tokens, tokensResult, ethBalance, chainName, prices);
   }
 
+  // temporary
+  const generateRandonDescription = () => {
```
Comment (turtlemoji) [inline src/utils/getVaultsData.ts:143]: `randon`

File: src/components/VaultCard.tsx [hunk 3]
```diff
@@ -143,3 +149,21 @@ const ChipsContainer = styled(Box)({
...
+const DescriptionContainer = styled('div')({
+  width: '100%',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  flexDirection: 'row',
+  marginTop: '0.8rem',
+  maxHeight: '6rem',
+  p: {
+    display: '-webkit-box',
```
[the remark this answers is not in the document]
Comment (0xArdy) [reply src/components/VaultCard.tsx:162]: yes

File: src/config/constants.ts [hunk 4]
```diff
@@ -8,6 +8,8 @@ const constants: Constants = {
 
   addresses: {
     AutomationVaultFactory: '0x49f06CfB7e9505b5a704Ca24E4674aa7B1A81D96',
+    // TODO: Update this address
```
[the remark this answers is not in the document]
Comment (0xArdy) [reply src/config/constants.ts:11]: yes, when the new contract is deployed https://github.com/defi-wonderland/xkeeper-core/pull/5

File: src/utils/getVaultsData.ts (hunk 1 above)
Comment (0xArdy) [reply src/utils/getVaultsData.ts:78]: yes, im waiting this https://github.com/defi-wonderland/xkeeper-core/pull/5

File: src/utils/getVaultsData.ts [hunk 5]
```diff
@@ -83,13 +98,14 @@ const fetchAndFormatData = async (
...
       { ...vaultContract, functionName: 'jobs' },
+      // { ...metadataContract, functionName: 'automationVaultMetadata' },
       ...balanceCalls,
     ],
   });
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  const [owner, name, relays, jobs] = multicallResult.slice(0, 4) as any;
-  const tokensResult = multicallResult.slice(4) as CallResult[];
+  const [owner, name, relays, jobs] = multicallResult.slice(0, 4) as any; // TODO: update to 5 when metadata is added
```
[the remark this answers is not in the document]
Comment (0xArdy) [reply src/utils/getVaultsData.ts:107]: same https://github.com/defi-wonderland/xkeeper-core/pull/5

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**